api/v1beta1/chaincode.go

```diff
@@ -34,3 +43,23 @@ func NextCond(instance *Chaincode) ChaincodeConditionType {
 	}
 	return condOrder[exp]
 }
+
+// GetChannel
+func (i *Chaincode) GetChannel(cli client.Client, cli2 controllerclient.Client) (*Channel, error) {
```

Comment:
If just want Get() function, just one "sigs.k8s.io/controller-runtime/pkg/client".Reader is enough.